api/utils/utils.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+)
+
+type server struct {
+	Address string `json:"address"`
+}
+
+type config struct {
+	Server server `json:"server"`
+}
+
+var Config config
```

Comment:
Hmm I don't think there's a need for this alias, you could just export your `Config` struct